src/sparsezoo/utils/onnx.py

```diff
@@ -42,6 +50,100 @@
 ]
 
 
+def save_onnx(
+    model: ModelProto, model_path: str, external_data_file: Optional[str] = None
+) -> bool:
+    """
+    Save model to the given path.
+
+    If the model's size is larger than the maximum protobuf size:
+        -   it will be saved with external data
+    If the model's size is smaller than the maximum protobuf size:
+        -   and the user nevertheless specifies 'external_data_file',
+            the model will be saved with external data.
+
+    :param model: The model to save.
+    :param model_path: The path to save the model to.
+    :param external_data_file: The optional name save the external data to.
+    :return True if the model was saved with external data, False otherwise.
+    """
+    if external_data_file is not None:
+        _LOGGER.info(f"Saving with external data: {external_data_file}")
+        onnx.save(
+            model,
+            model_path,
+            save_as_external_data=True,
+            all_tensors_to_one_file=True,
+            location=external_data_file,
+        )
+        return True
+
+    if model.ByteSize() > onnx.checker.MAXIMUM_PROTOBUF:
+        _LOGGER.info(
+            "The ONNX model is too large to be saved as a single protobuf."
+            "Saving with external data... "
+        )
+
+        _LOGGER.info(f"Saving with external data: {external_data_file}")
```

Comment:
Wouldn't `external_data_file` evaluate to `None` here?

Reply:
You are right, good catch. Will just remove this line of code.